src/functionalities/OutputNode.js

```diff
@@ -49,6 +49,8 @@ class OutputNode extends Functionality {
     }
   }
 
+  get isOutputNode() { return this.type === OutputNode.TYPE }
```

Comment:
why is this even required? The type of a OutputNode cannot be anything else but `OutputNode`

Reply:
100% right - this was just me losing my way in the weeds of thinking about inheritance
I will update!